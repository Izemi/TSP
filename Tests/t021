#!/bin/bash
# Nearest with ties

trap "/usr/bin/killall -q -u $USER ./TSP 2>/dev/null" 0 1 2 3 9 15
trap "/bin/rm -f $STDERR" 0 1 2 3 9 15
if [ ! -x ./TSP ]; then
  echo './TSP is missing or not executable'
  echo './TSP is missing or not executable' 1>&2
  exit 1
fi


$BIN/run -stdout=stdout.out -stderr=stderr.out ./TSP $WHERE/t021.in -nearest E00 EE1 EE2 EW1 EW2 NE1 NE2 SW1 SW2 < /dev/null
cat stdout.out stderr.out | cut -b 1-24
