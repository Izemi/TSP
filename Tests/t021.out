-nearest  :     15432.59
